### DISCUSSION TITLE
피그마에 존재하는 아이콘을 개발단에 빠르게 적용하기 (피그마 플러그인, 아이콘 라이브러리) - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#52] [Announcements]
# 피그마에 존재하는 아이콘을 개발단에 빠르게 적용하기 (피그마 플러그인, 아이콘 라이브러리) - 정현수 기술 블로그

피그마 플러그인을 통해서 아이콘을 깃허브에 배포하고, 개발단에서 바로 사용할 수 있게한 경험을 공유합니다.

https://junghyeonsu.com/posts/quickly-apply-icons-that-exist-in-figma-to-your-dev-team/

**Comment by Henry-Hong** [comment]
@icona/generator: Icona asset generator 코드를 좀 살펴보고 싶었는데, 
지금은 404 NOT FOUND 네용.

사내 도구로 사용중이라서 그런가요?
